element-hq/element-x-android
# Timeline related improvements

## Content
Two changes in this PR, related to timeline :  
- Pinned timeline items doesn't need debounce anymore 
- Regular timeline were briefly emitting empty list when navigating back


## Motivation and context



## Screenshots / GIFs



## Tests



- Open room
- Scroll backward in the timeline
- Open room details
- Press back
- See timeline kept his position

## Tested devices

- OS version(s):

## Checklist
